joindinapp/joindinapp-Info.plist

```diff
@@ -37,13 +37,24 @@
 	<string>1.6</string>
 	<key>LSRequiresIPhoneOS</key>
 	<true/>
+	<key>NSAppTransportSecurity</key>
+	<dict>
+		<key>NSExceptionDomains</key>
```

Comment:
It's [HTTPS](https://github.com/richsage/joind.in-iphone/blob/ios9/joindinapp/joindinapp-Info.plist#L54) at the moment anyway, but the certificate is SHA1 so iOS 9 rejects it (as far as I can tell anyway)